README.md

```diff
@@ -544,6 +544,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [John Proodian](https://github.com/johnproodian)
   - [Jome Favourite](https://github.com/jomefavourite)
   - [Jonathan Niklasson Godar](https://github.com/JonathanGodar)
+  - [Jonathan Llemit Jr.](https://github.com/jcljr00)
```

Comment:
Please add your name in line 546, after Jome Favourite.


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
done